cmd/internal/cli/search.go

```diff
@@ -16,8 +16,8 @@ import (
 	"github.com/apptainer/apptainer/internal/pkg/client/library"
 	"github.com/apptainer/apptainer/pkg/cmdline"
 	"github.com/apptainer/apptainer/pkg/sylog"
+	"github.com/apptainer/container-library-client/client"
```

Comment:
@DrDaveD for consistency, if one import has a name override, all of the imports in different files should have that same name override.

at the moment, some files have `client.` while others have `libClient.`

Reply:
I think that's outside the scope of this PR.  If that's to be a consistent policy, someone should examine the whole codebase for that.  Make an issue for that if you lie.